### DISCUSSION TITLE
请问Wondertrader支持卡方科技的下单程序吗？

### DISCUSSION THREAD

**Discussion by Guan2019310358** [wondertrader/wondertrader#207] [Q&A]

**Comment by Billy1900** [comment]
卡方应该是order发到卡方再拆单吧，所以应该是支持的，只是需要自己改写

**Comment by Guan2019310358** [comment]
哇这么晚还回复，感激不尽啊！！！



Xuyang&nbsp;Guan
***@***.***







                       
Original Email
                       
                     

Sender:"Yangtze"< ***@***.*** &gt;;

Sent Time:2023/10/3 21:04

To:"wondertrader/wondertrader"< ***@***.*** &gt;;

Cc recipient:"Guan2019310358"< ***@***.*** &gt;;"Author"< ***@***.*** &gt;;

Subject:Re: [wondertrader/wondertrader] 请问Wondertrader支持卡方科技的下单程序吗？ (Issue #204)




 
卡方应该是order发到卡方再拆单吧，所以应该是支持的，只是需要自己改写

**Comment by wondertrader** [comment]
wondertrader内置了XTP的algo算法总线，通过这个算法总线，可以调用很多不同的算法

**Comment by wondertrader** [comment]
咨询的话，不要放到issue吧

**Comment by Guan2019310358** [comment]
抱歉，我这就删除



Xuyang&nbsp;Guan
***@***.***







                       
Original Email
                       
                     

Sender:"wondertrader"< ***@***.*** &gt;;

Sent Time:2023/10/11 16:28

To:"wondertrader/wondertrader"< ***@***.*** &gt;;

Cc recipient:"Jason"< ***@***.*** &gt;;"Author"< ***@***.*** &gt;;

Subject:Re: [wondertrader/wondertrader] 请问Wondertrader支持卡方科技的下单程序吗？ (Issue #204)




 
咨询的话，不要放到issue吧
